lib/graphql/mutations.js

```diff
@@ -2,6 +2,8 @@ import { graphql } from 'react-apollo';
 import gql from 'graphql-tag';
 import { pick, isArray } from 'lodash';
 
+const gqlV2 = gql; // Needed for lint validation of api v2 schema.
```

Comment:
Hmm, have you tried running `npm run graphql:update`?

Reply:
I have done - I followed this convention from another file using a graphql V2 mutation. Maybe @znarf can comment whether it's still necessary?

Reply:
@kewitz it's not related to that, it's for eslint to know that it should validate against the API V2's schema.

You could import the helper from here for more clarity: https://github.com/opencollective/opencollective-frontend/blob/9490073fb8de9d24e38190a5b921d7e991b6bbf9/lib/graphql/helpers.js#L4

Reply:
Oh, noted!